scripts/backfill-project-areas.sh

```
@@ -0,0 +1,395 @@
+    --help|-h)
+      echo "Usage: $0 [--apply] [--all] [--strip-labels]"
+      echo "  --apply         Apply changes (default is dry run)"
+      echo "  --all           Re-evaluate all items, not just unset ones"
+      echo "  --strip-labels  Remove area:* labels from issues (keeps them on PRs)"
+      exit 0
+      ;;
+    *) echo "Unknown arg: $arg"; exit 1 ;;
+  esac
+done
+
+AREA_LABELS="area:core area:agents area:dashboard area:cli area:server area:documentation"
```

Shellcheck (0.11.0)

[warning] AREA_LABELS appears unused. Verify use (or export if used externally).

(SC2034)